app/services/geek-jokes.js

```diff
@@ -0,0 +1,21 @@
+const axios = require('axios');
+
+const getJoke = () => {
+  axios
+    .get('https://geek-jokes.sameerkumar.website/api?format=json')
+    .then(
+      response =>
+        // handle success
+        response.data.joke
+    )
+    .catch(
+      error =>
+        // handle error
+        error
```

Comment:
Return a specific error here.
Also, you can use informative logs to the app registry